--- Mail-1.1.14/usr/share/pear/Mail.php	2007-07-04 00:44:25.993319232 +0300
+++ Mail-1.1.14/usr/share/pear/Mail.php	2007-07-04 00:45:15.534454909 +0300
@@ -151,7 +151,7 @@
         foreach ($headers as $key => $value) {
             if (strcasecmp($key, 'From') === 0) {
                 include_once 'Mail/RFC822.php';
-                $parser = &new Mail_RFC822();
+                $parser = new Mail_RFC822();
                 $addresses = $parser->parseAddressList($value, 'localhost', false);
                 if (PEAR::isError($addresses)) {
                     return $addresses;
